### ISSUE TITLE
Upstream: a11y(critical) authoring course import drop zone has no label

### ISSUE THREAD

**Issue by bmtcril** [openedx/end-to-end-tests#57]
https://github.com/openedx/frontend-app-authoring/issues/3241
